conans/test/integration/command_v2/list_test.py

```diff
@@ -205,12 +226,35 @@ def test_list_latest_package_revisions(self):
         Local Cache:
           test_recipe
             test_recipe/1.0.0@user/channel#ddfadce26d00a560850eb8767fe76ae4 .*
-              PID: da39a3ee5e6b4b0d3255bfef95601890afd80709 .*
-                PREV: 9c929aed65f04337a4143311d72fc897
+              PID: da39a3ee5e6b4b0d3255bfef95601890afd80709
+                PREV: 9c929aed65f04337a4143311d72fc897 .*
+        """)
+        assert bool(re.match(expected_output, str(self.client.out), re.MULTILINE))
+
+    def test_list_all_the_latest_package_revisions(self):
+        self.client.save({
+            "hello.py": GenConanfile("hello", "1.0.0").with_package_file("file.h", "0.1"),
+            "bye.py": GenConanfile("bye", "1.0.0").with_package_file("file.h", "0.1")
+        })
+        self.client.run("create hello.py --user=user --channel=channel")
+        self.client.run("create hello.py --user=user --channel=channel")  # latest prev
+        self.client.run("create bye.py --user=user --channel=channel")
+        self.client.run("create bye.py --user=user --channel=channel")   # latest prev
```

Comment:
@memsharded Actually, it's passing, but it's not creating several PREVs. I know that using the command `conan new xxxx` and several `conan create .` will work as expected but it's too slow, so can I improve it?